src/index.html

```diff
@@ -2,14 +2,339 @@
 <html lang="ru">
 
 <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=р1, initial-scale=1.0">
-    <title>Главная</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=р1, initial-scale=1.0">
```

Comment:
Что такое `width=р1`? Кажется это было еще в шаблоне. Но я бы рекомендовал поменять на 

```suggestion
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
```